_includes/sections/instant-messenger.html

```diff
@@ -40,44 +67,199 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
+    </ul>
+
+    <h4>Disadvantages</h4>
+    <ul>
+        <li>More complex to develop new features because standards documents need to be written to ensure a thought out implementation.</li>
+        <li>Some metadata may be available. Who is talking to whom but not what is being said if E2EE (end-to-end encryption) is used.</li>
+    </ul>
+</div>
+
+{%
+  include cardv2.html
+  title="Matrix"
+  image="/assets/img/tools/Riot.png"
+  description="<p><a href=\"https://matrix.org/docs/guides/introduction\">Matrix</a> is an open source project that publishes the <a href=\"https://matrix.org/docs/spec\">Matrix open standard</a> for secure, decentralized, real-time communication. Matrix also has <a href=\"https://matrix.org/docs/guides/types-of-bridging.html\">bridging functionality</a>, allowing you to connect to other chat protocols such as IRC or Telegram.</p>
+  <p><a href=\"https://en.wikipedia.org/wiki/Riot.im\">Riot.im</a> is the popular reference client produced by the Matrix.org team. It offers optional E2EE (end-to-end encryption) for 1:1 and group conversations that <strong>must</strong> be turned on by the user. (This can be done by clicking on the toggle switch which is accessed by clicking the room name or user name of the chat → Security & Privacy → Encrypted). In the future it will be <a href=\"https://github.com/vector-im/riot-web/issues/6779\">on by default.</a></p>"
+  website="https://about.riot.im/"
+  forum="https://forum.privacytools.io/t/discussion-riot-im/665/"
+  github="https://github.com/vector-im/riot-web/"
+  windows="https://riot.im/download/desktop/"
+  mac="https://riot.im/download/desktop/"
+  linux="https://riot.im/download/desktop/"
+  fdroid="https://f-droid.org/repository/browse/?fdid=im.vector.alpha"
+  googleplay="https://play.google.com/store/apps/details?id=im.vector.app"
+  ios="https://itunes.apple.com/us/app/vector.im/id1083446067"
+  web="https://riot.im/app/"
+%}
+
+{%
+  include cardv2.html
+  title="XMPP"
+  image="/assets/img/tools/XMPP.png"
+  description="<p><a href=\"https://xmpp.org/about\">XMPP</a> (Extensible Messaging and Presence Protocol) is an open source communications protocol that began development in 1999. Since then XMPP has been extended by the publishing of XEPs (XMPP Extension Protocols).</p>
+  <a href=\"https://conversations.im/omemo/\">OMEMO</a> is the most popular XEP for E2EE (end-to-end encryption). Unlike Matrix, clients are only developed by the community and not by the foundation itself. We've linked some of the ones which we think are good choices below."
+  labels="warning:Warning inconsistent E2EE:VOIP and File transfers may not be end-to-end encrypted"
+  website="https://xmpp.org/"
+  forum=""
+  windows="https://gajim.org/"
+  mac="https://gajim.org/"
+  linux="https://gajim.org/"
```

Comment:
```suggestion
  linux="https://dino.im/"
```

This may be controversial though as it's still in early development, while being the most modern. It's also horribly outdated in some repos.

Reply:
Oh, Gajim doesn't have OMEMO by default and needs a plugin for that.

Reply:
I do like the look of Dino, so I would be keen to approve that however there doesn't appear to be any recently tagged releases: https://github.com/dino/dino/releases this maybe why it's horribly outdated in some repositories.

Also nobody brought this up in https://github.com/privacytoolsIO/privacytools.io/issues/1377